Code/autopkglib/MunkiImporter.py

```diff
@@ -539,23 +577,44 @@ def main(self):
             return
 
         # copy pkg/dmg to repo
-        relative_path = self.copy_item_to_repo(pkginfo)
+        uploaded_pkgpath = munkiimportlib.copy_item_to_repo(
+            repo,
+            self.env["pkg_path"],
+            pkginfo.get("version"),
+            self.env.get("repo_subdirectory", ""),
```

Comment:
This isn't technically necessary as the default value for this argument is '':
https://github.com/munki/munki/blob/f1b5162841475a9486983faabff22504e14c06ae/code/client/munkilib/admin/munkiimportlib.py#L46